package/kernel/qca-ssdk/patches/0001-SSDK-config-add-kernel-5.10.patch

```diff
@@ -1,56 +0,0 @@
-From 6ee395e1cab32d1d4899d7846ea99e6e2dc0a9e4 Mon Sep 17 00:00:00 2001
```

Comment:
Need to refresh`0005-SSDK-config-add-kernel-5.15.patch` as it is built after this patch.

Reply:
Indeed. Thanks!